lisa/analysis/tasks.py

```diff
@@ -423,6 +424,34 @@ def df_task_total_residency(self, task):
 
         return residency_df
 
+    @df_task_states.used_events
+    def df_tasks_total_residency(self, tasks, ascending=False):
+        """
+        DataFrame of tasks execution time on each CPU
+
+        :param tasks: a list of tasks to report
+        :type tasks: list(int) or list(str) or a mix of the two
+
+        :param ascending: Set True to plot before short running tasks,
+                          False by deafult
+        :type ascending: bool
+        """
+        res_df = pd.DataFrame()
+        for task in tasks:
+            if isinstance(task, str):
+                pid = self.trace.get_task_by_name(task)
```

Comment:
In android the same task name might appear under several pids, so the returned pid here will not be a single item

Reply:
Yes, that's true... The only thing we can do in that case is to plot add all the PIDs... which is not covered yet... :/